c2corg_ui/static/partials/feedtable.html

```diff
@@ -24,12 +24,11 @@
           ng-init="type = wfeedCtrl.getDocumentType(doc.document.type);">
 
         <td class="text-left">
-          <span ng-if="doc['document']['version'] > 1 && doc['document']['type'] !== 'i'">
-            <a ng-href="{{wfeedCtrl.getVersionHref(doc)}}">{{::doc['document']['title']}}</a>
-          </span>
-          <span ng-if="doc['document']['version'] == 1 || doc['document']['type'] == 'i'">{{::doc['document']['title']}}</span>
-          <span ng-if="doc['document']['title'].length == 0" translate>Untitled</span>
-          <span ng-if="doc['lang'] != locale.lang">({{::doc['lang']}})</span>
+          <a ng-href="{{wfeedCtrl.getVersionUrl(doc)}}">
+            <span ng-if="doc['document']['title']">{{::doc['document']['title']}}</span>
+            <span ng-if="!doc['document']['title']" translate>Untitled</span>
```

Comment:
> I checked and there is again that problem with not-accessible image versions.

That's not our business here. It's an API issue: https://github.com/c2corg/v6_ui/issues/1503
Hopefully it will be fixed and the links will work here as well. But it does not make sense to make workarounds here, it's better to fix the problem at its source.